test-e2e/expand.rs

```diff
@@ -0,0 +1,1994 @@
+#![feature(prelude_import)]
+#[prelude_import]
+use std::prelude::rust_2021::*;
+#[macro_use]
+extern crate std;
+extern crate core;
```

Comment:
not sure why there is the file?

Reply:
My fault while rebasing branch. those two files should not be included. (I created the file for checking the macro result)
I removed them at [e3b5198](https://github.com/zkmopro/mopro/pull/247/commits/e3b51989d8ad4d07f7b4dfa154d23d01e42e2d2d)